### PR TITLE
Expose new enums for VPNGatewaySku for VPN Gateways

### PR DESCRIPTION
**Pull request by ritwikbasu**
Expose new enums for VPNGatewaySku for VPN Gateways

### CODE REVIEW

**Comment by markcowl** [comment]
@ritwikbasu This is out of date with the branch, please pull the latest

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs [hunk 1]
```diff
@@ -21,6 +21,9 @@ public static class VirtualNetworkGatewaySkuTier
         public const string HighPerformance = "HighPerformance";
         public const string Standard = "Standard";
         public const string UltraPerformance = "UltraPerformance";
+        public const string VpnGw1 = "VpnGw1";
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs]: @ritwikbasu can you link the swagger spec for this change?
Also what is still not clear is, how come the sdk branch as 9.1.0-preview as the latest version of network package, but what is published out on nuget is 10.0.0-preview?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs (hunk 1 above)
Comment (ritwikbasu) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs]: This is the swagger PR : https://github.com/Azure/azure-rest-api-specs/pull/1130.

I pulled the latest sdk branch - I am not sure of the versioning. I did not change anything except for the three generated code files in the PR.

Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs]: @ritwikbasu in order to publish this SDK, it has to be a different version that is already not published to nuget.
But also, if currently your SDK that is published on nuget as 10.0, but you end up publishing this particular with 9.1, that will create confusion.
this version should be 10.1.0

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
@ritwikbasu I don't see any version information.
What version will be this nuget? I see now that from prior iteration, you have completely ignored version number, what will be the version number of your nuget that you plan to publish.

**Comment by ritwikbasu** [comment]
Strange – I pushed a new commit to update the version information:

[cid:image002.jpg@01D2BDC4.4A200DA0]

**Comment by shawns1** [comment]
@azuresdkci please test

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**